lvmd/device_class_manager.go

```diff
@@ -36,6 +51,10 @@ type DeviceClass struct {
 	StripeSize string `json:"stripe-size"`
 	// LVCreateOptions are extra arguments to pass to lvcreate
 	LVCreateOptions []string `json:"lvcreate-options"`
+	// Type is the name of logical volume target, supports 'thick' (default) or 'thin' currently
+	Type LVType `json:"type,omitempty"`
+	// ThinPoolConfig holds the configuration for thinpool in this volume group corresponding to the device-class
+	ThinPoolConfig *ThinPoolConfig `json:"thin-pool,omitempty"`
```

Comment:
```suggestion
	// Type is the name of logical volume target, supports 'thick' (default) or 'thin' currently
	Type LVType `json:"type"`
	// ThinPoolConfig holds the configuration for thinpool in this volume group corresponding to the device-class
	ThinPoolConfig *ThinPoolConfig `json:"thin-pool"`
```

Reply:
- ack, we might get null values, is that fine?

Reply:
IMO, `omitempty` option effects when converting this struct to string. I guess there is no such code.